R/derive_param_clinbenefit.R

```diff
@@ -0,0 +1,256 @@
+#' Derive a Clinical Benefit Parameter
+#'
+#' Add a clinical benefit/disease control parameter to the input dataset.
+#'
+#' @details
+#' Clinical benefit/disease control is first identified for looking for subjects
+#' having objective response, and then derived for subjects that have at least
```

Comment:
i would take out "objective" before response everywhere here - i dont think we used that terminology anywhere that i saw. just leaving as response maybe leaves more open for whatever companies decide to input here

Reply:
roger that